test/fileImport.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+const _ = require('lodash');
+const config = require('config');
+const should = require('should');
+const homedir = require('homedir');
+
+const client = require('./lib/client.js').client;
+const nav = require('./lib/nav/navigation.js')(client, config);
+const micis = require('./lib/auth/micis.js')(client);
+
+describe('Doing various imports', function() {
+    this.timeout(config.defaultTimeout);
+
+    before('initialize', function(done) {
+        client.clientReady.then(function boot() {
+            if (!micis.loggedOn) {
+                micis.logon();
+            }
+            client.call(done);
+        });
+    });
+
+    describe('Import subjects', function(done) {
```

Comment:
Note: Import subjects and import assessments were not automated as currently they are not ideally functioned.  For example, when user first logs in to micis, go to import subjects, if s/he clicks on button "choose and upload files", there will be no responses. Instead, s/he needs to click on `show site ids` button first to activate some kinds of functionality, then click `choose and upload files` to open up the file selection windows. This has been noted long time ago (and mgmt knows), but put on the back burner for the fix. I think this should be fixed before making the test automated.